Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -412,23 +412,43 @@ final class BuildCommandTests: CommandsTestCase {
         }
     }
 
-    func testXcodeBuildSystemDefaultSettings() async throws {
-        #if !os(macOS)
-        try XCTSkipIf(true, "test requires `xcbuild` and is therefore only supported on macOS")
-        #endif
+    private func testBuildSystemDefaultSettings(buildsystem: String) async throws {
```

Comment:
nit: inconsistent capitalization
```suggestion
    private func testBuildSystemDefaultSettings(buildSystem: String) async throws {
```